### ISSUE TITLE
Possibility to get telemetry data of surrounding traffic vehicles?

### ISSUE THREAD

**Issue by manuelschmidt** [nlhans/ets2-sdk-plugin#41]
Hey,

thanks for your great work here! I also really enjoyed watching for demos on youtube - you did some impressive work with ETS2.

As far as I understand, the telemetry data is only available for the ego-vehicle, is that correct?
I wonder if there is still some chance to get the telemetry data for all the other surrounding vehicles somehow? (Similar to what you did for implementing ACC)
Any chance to customize the SDK for also providing that information?

Best regards

Manuel

**Comment by nlhans** [comment]
No unfortunately there is no support for this current version of ETS2, nor by the SDK plugin.

A previous implementation ran alongside (not inside as a plugin) ETS2 and was only working for version 1.19. The implementation required knowledge of some absolute memory addresses of the game, which unfortunately changed every time the game executable is patched. Although there are some ways around that (which I tried), ultimately it did break in a few patches. 

Basically it's the same struggles ETS2MP is facing when games are updated. Unfortunately, as you may judge by this late response, my time does not permit to be on top of these developments.
